format-code.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+find firmware/common firmware/baseband firmware/application firmware/test/application firmware/test/baseband -iname '*.h' -o -iname '*.hpp' -o -iname '*.cpp' -o -iname '*.c' | xargs clang-format -style=file -i
```

Comment:
You need to specify version 13, or this will not work correctly clang-format-13

Reply:
Please see the correct command here https://github.com/eried/portapack-mayhem/wiki/Code-formatting